integration/kube_integration_test.go

```diff
@@ -1928,6 +1928,8 @@ func newNamespace(name string) *v1.Namespace {
 	}
 }
 
+const localPodImage = "alpine-webserver:v1"
```

Comment:
Please add a comment here explaining where the image comes from. Maybe even link the workflow so folks know how to reproduce it locally.